man/rmd/recipes.Rmd

```diff
@@ -22,13 +22,13 @@ There are two different types of operations that can be sequentially added to a
 
 - **Checks** are operations that conduct specific tests of the data. When the test is satisfied, the data are returned without issue or modification. Otherwise, an error is thrown.
 
-If you have defined a recipe and want to see which steps are included, use the `tidy()` method on the recipe object.
+If you have defined a recipe and want to see which steps are included, use the [tidy()] method on the recipe object.
```

Comment:
```suggestion
If you have defined a recipe and want to see which steps are included, use the [`tidy()`][tidy.recipe()] method on the recipe object.
```
Otherwise it links to something confusing